docs/templates/model-serving/project.md

```diff
@@ -0,0 +1,90 @@
+# Project/Connection
+
+In Hopsworks [a Project is a sandboxed set](https://www.logicalclocks.com/blog/how-we-secure-your-data-with-hopsworks) of users, data, and programs (where data can be shared in a controlled manner between projects).
+
+Each Project can have its own Model Registry and Model Serving. However, it is possible to [share Model Registry](#sharing-a-model-registry) and Model Serving among projects.
+
+When working with the Model Registry & Serving from a programming [environment](../setup.md) you can connect to a single Hopsworks instance at a time, but it is possible to access multiple Model Registries and Model Servings simultaneously.
+
+A connection to a Hopsworks instance is represented by a [`Connection` object](#connection). Its main purpose is to retrieve the API Key if you are connecting from an external environment.
```

Comment:
remove this - 
"Its main purpose is to retrieve the API Key if you are connecting from an external environment."